examples/demo-apps/android/LlamaDemo/android-llm-device-farm-test-spec.yml

```diff
@@ -73,8 +73,16 @@ phases:
           fi
         fi;
 
+      # Run the new generic benchmark activity
+      - echo "Run LLM benchmark"
+      - |
+        adb -s $DEVICEFARM_DEVICE_UDID shell am start -n com.example.executorchllamademo/.Benchmarking \
+        --es "model_dir" "/data/local/tmp/llama" \
+        --es "tokenizer_path" "/data/local/tmp/llama/tokenizer.bin"
```

Comment:
Thank you! Exactly what i want to do :D 

Need to find a way to wait for the file to result file to appear. shell am start is async

Reply:
Oh, got it. TIL.  I'm still working on this to make the command works, so stay tuned :)

Reply:
Does this kind of stuff work? 🤔 
adb shell while [ ! -f /data/local/tmp/result.txt ]; do sleep 1; done

Reply:
Yeah, it's working, but adb shell doesn't like the way I write bash script, so I need to look for a work around by cat-ing the results.  It works nonetheless, so I guess we are good :)